src/Raven.Server/Smuggler/DatabaseDataExporter.cs

```diff
@@ -30,16 +33,20 @@ public ExportResult Export(DocumentsOperationContext context, string destination
 
         public ExportResult Export(DocumentsOperationContext context, Stream destinationStream)
         {
-            long lastDocsEtag = 0;
+            var result = new ExportResult();
 
             using (var gZipStream = new GZipStream(destinationStream, CompressionMode.Compress, leaveOpen: true))
             using (var writer = new BlittableJsonTextWriter(context, gZipStream))
             {
                 writer.WriteStartObject();
 
+                writer.WritePropertyName(context.GetLazyString("BuildVersion"));
+                writer.WriteInteger(40000);
```

Comment:
Unless there is a REALLY good reason not to do this, we want to be able to import 4.0 to 3.5